code/obj/machinery/singularity.dm

```diff
@@ -79,8 +79,8 @@ Contains:
 //////////////////////////////////////////////////// Singularity /////////////////////////////
 
 /obj/machinery/the_singularity/
-	name = "gravitational singularity"
-	desc = "A Gravitational Singularity."
+	name = "Gravitational Singularity"
```

Comment:
No reason to capitalise the name. it's not a proper noun and it's inconsistent when examining as that's what shows up when you examine an item EG : "It's a Gravitational Singuliarty" when the correct form would be "It's a gravitational singuliarty"

Reply:
Done, thanks for the feedback!